test/inductor/test_cuda_repro.py

```diff
@@ -1204,6 +1204,46 @@ def forward(self, x):
             out2 = m(input_tensor)
             self.assertEqual(out, out2, atol=1e-3, rtol=1e-3)
 
+    def test_int64_index_intermediate(self):
+        def foo(split_1):
+            view_23 = torch.ops.aten.view.default(inp, [-1, 8192, 8192])
+            split_1 = torch.ops.aten.split.Tensor(view_23, 1024, 1)
```

Comment:
Clean up this a bit?

`split_1` argument is not used and being overwritten directly.